WordPress/src/main/java/org/wordpress/android/ui/accounts/AbstractFragment.java

```diff
@@ -63,12 +63,12 @@ protected boolean onDoneEvent(int actionId, KeyEvent event) {
         return false;
     }
 
-    protected boolean didPressNextKey(int actionId, KeyEvent event) {
+    public static boolean didPressNextKey(int actionId, KeyEvent event) {
         return actionId == EditorInfo.IME_ACTION_NEXT || event != null && (event.getAction() == KeyEvent.ACTION_DOWN
                 && event.getKeyCode() == KeyEvent.KEYCODE_NAVIGATE_NEXT);
```

Comment:
I know this is not in the changeset, but AS is complaining that `KeyEvent.KEYCODE_NAVIGATE_NEXT` requires `API 23`. Since the current feature is the only one using this method, I thought it's worth mentioning here. I am not sure if this affects us at all. I can check if you want, just let me know.